data-structures-queues/postpone.js

```diff
@@ -0,0 +1,6 @@
+export default function postpone(queue) {
+  if (queue.peek() !== undefined) {
+    const front = queue.dequeue();
```

Comment:
You could eliminate this local variable by just passing `queue.dequeue()` directly.